src/components/Reviews/Reviews.jsx

```diff
@@ -9,31 +9,70 @@ export default function Reviews() {
     const [loading, setLoading] = useState(true);
     const [reviewsList, setReviewsList] = useState([]);
     const { category } = useParams();
+    const [sortBy, setSortBy] = useState("date");
+    const [orderBy, setOrderBy] = useState("asc");
+
 
     useEffect(() => {
         setLoading(true);
 
-        getReviews(category).then((response) => {
+        getReviews(category, null, sortBy, orderBy).then((response) => {
             setLoading(false);
             return setReviewsList(response.reviews);
         });
-    }, [category])
+    }, [category, sortBy, orderBy])
+
     
     if (loading) {
         return (<h2>Loading...</h2>)
     } else {
         return (
+          <section>
+            <label htmlFor="sortBy">Sort By: </label>
+            <select
+              id="sortBy"
+              onChange={(e) => setSortBy(e.target.value)}
+              value={sortBy}
+            >
+              <option key="date" value="date">
+                Date
+              </option>
+              <option key="votes" value="votes">
+                Votes
+                    </option>
+                    {/* Comment count hasn't been implemented in the back end. */}
+              {/* <option key="comment_count" value="comment_count">
+                Comment Count
+              </option> */}
+                </select>
+                
+            <label htmlFor="orderBy">Order By: </label>
+            <select
+              id="sortBy"
+              onChange={(e) => setOrderBy(e.target.value)}
+              value={orderBy}
+            >
+              <option key="ascending" value="asc">
+                Ascending
+              </option>
+              <option key="descending" value="desc">
+                Descending
+              </option>
+            </select>
```

Comment:
Becasue your dropdowns and list are in the same component, you re-render the drop downs and list when the state chenges, might be worth extrapolating the list into it's own component so the dropdown doesn't re-render